space-concordia-robotics/robotics-prototype
# [#586] Bring back files from purge that turned out to be useful after…

# Assignee Section

## Description
Brought back useful files that were deleted in purge and make fixes to environment setup that accounts for changes made from purge. Tests directory was brought back as there were actual unit tests in there that were useful. Build failures for tests should be addressed in a seperate issue rather than removing all of the tests.

### Steps for Testing
Look over environment setup and confirm that all files under there are in proper locations and are present in repo. Including the additions to ~/.bashrc and ~/.bash_aliases.

closes #586 

The approval from all software team leads is necessary before merging.

# Reviewer Section

Aside from local testing and the General Integration Test it is implied that static analysis should be included in the verification process.


For Pull Requests that do not include code changes, it is not required to perform the tests above.
